### ISSUE TITLE
Unknown modules opened

### ISSUE THREAD

**Issue by Xerus2000** [sghpjuikit/player#166]
I get an array of Warnings when I start the application:
```
WARNING: Unknown module: javafx.controls specified to --add-exports
WARNING: Unknown module: javafx.graphics specified to --add-exports
WARNING: Unknown module: javafx.graphics specified to --add-opens
WARNING: Unknown module: javafx.graphics specified to --add-opens
WARNING: Unknown module: javafx.web specified to --add-opens
WARNING: Unknown module: javafx.controls specified to --add-opens
WARNING: Unknown module: javafx.graphics specified to --add-opens
WARNING: Unknown module: javafx.graphics specified to --add-opens
WARNING: Unknown module: javafx.controls specified to --add-opens
```

**Comment by sghpjuikit** [comment]
Yes, I wrote about this on Zulip. I'd like to use Jigsaw kill switch for this.

**Comment by sghpjuikit** [comment]
This may be because javafx would like to have its modules on modelupath instead of classpath.

**Comment by sghpjuikit** [comment]
Bookmarks:
https://stackoverflow.com/questions/46084751/how-to-express-dependency-in-maven-on-java-ee-features-for-transition-to-java-9/46086920#46086920
https://github.com/openjfx/javafx-gradle-plugin/blob/master/src/main/java/org/openjfx/gradle/tasks/ExecTask.java

**Comment by sghpjuikit** [comment]
- I have removed all `add-exports` flags, by rewriting all code that relied on inaccessible classes. Finally, after 3 years, the Jigsaw curse is gone.
- As for `--add-opens`, these won't go that easily, so some of the warnings are still present.

Moreover
- I have also tried to add the required javafx modules, but I was unable to make it work once again (the modules could not be found). ButI really do not want to use that plugin!
- `--illegal-access=permit` will also not help (its on by default) as it only affects reflecting on the core modules, which javafx is, sadly, not part of anymore

**Comment by sghpjuikit** [comment]
Most of the flags are because of the XStream library, which will be removed shortly.

**Comment by sghpjuikit** [comment]
Down to 4 warnings. All related to custom workarounds for complicated design issues or internal javafx problems.

**Comment by sghpjuikit** [comment]
Fixed.

I removed all `--add-opens`.  I know of at least 1 use which should still complain of illegal reflective access, but oddly enough - it no longer happens. Either way - fixed in 6616166
